packages/base-styles/_colors.scss

```diff
@@ -5,6 +5,7 @@
 // WordPress grays.
 $black: #000;			// Use only when you truly need pure black. For UI, use $gray-900.
 $gray-900: #1e1e1e;
+$gray-800: #484848;
```

Comment:
Why re-introduce one of the grays?

Reply:
sorry, do you mean it's already defined because it was removed before? I added to try to follow the color of the current implementation of the calendar in the post schedule component. Happy to change to another color if it's required. :-)

Reply:
We're trying to reduce the number of colors used in the editor UI, so using one of the existing shades is definitely recommended over reintroducing one that was removed.

Reply:
See also: https://make.wordpress.org/design/2020/09/18/project-reducing-colors-in-core/

Reply:
thanks!